notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>isn't it 7 features? 9 after feature importance -2 after OOB analysis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>good catch!</p>